awstesting/behavior_tests/sampleresetxmlprotocolservice/behav_test.go

```diff
@@ -0,0 +1,7351 @@
+// Code generated by private/model/cli/gen-api/main.go. DO NOT EDIT.
+
+// +build go1.10,integration
+
+package sampleresetxmlprotocolservice_test
+
+import (
+	"bytes"
+	"encoding/base64"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/corehandlers"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/request"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/awstesting"
+	"github.com/aws/aws-sdk-go/awstesting/behavior_tests/sampleresetxmlprotocolservice"
+	"github.com/aws/aws-sdk-go/private/util"
+	"github.com/google/go-cmp/cmp"
+)
+
+func parseTime(layout, value string) *time.Time {
+	t, err := time.Parse(layout, value)
+	if err != nil {
+		panic(err)
+	}
+	return &t
+}
+
+func assertRequestMethodEquals(t *testing.T, req *request.Request, val string) bool {
```

Comment:
I think all of these asserts should be in a shared `behaviortesting` package instead of regenerating these into each behavior_test file.